naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/resolvers/NaptimeResolver.scala

```diff
@@ -143,16 +143,8 @@ class NaptimeResolver extends DeferredResolver[SangriaGraphQlContext] with Stric
               }.toMap
               sourceRequests.map { sourceRequest =>
                 // TODO(bryan): Clean this up
-                val ids = sourceRequest.arguments
-                  .find(_._1 == "ids")
-                  .map(_._2)
-                  .map {
-                    case JsArray(idValues) => idValues
-                    case value: JsValue => List(value)
-                  }
-                  .getOrElse(List.empty)
-
-                val elements = ids.flatMap(parsedElementsMap.get).toList
+                val elements = parseAndDeduplicateIds(sourceRequest)
```

Comment:
can we avoid doing this and rename `mergeMultigetRequests` to `mergeMultigetRequestsAndDedupIds`? That way readers don't have to wonder why there's dedup logic in 2 places.